### ISSUE TITLE
Search Capabilities in RFI and general.

### ISSUE THREAD

**Issue by amit-k86** [OrderMail/blacklabel#22]
Acceptance Criteria:-
1. Search Capabilities to be on Dashboard & on RFI (Same will be in future on PO, SO etc.)
2. In Dashboard, After user enters first four alphabets, Auto Suggest to work. Once Selected, Then Auto. Primary Contact Name & Email displayed.
3. In RFI, Once User enters upto three alphabets, Auto Suggest to work in only his contacts (Search not activated). After four alphabets, Auto Suggest to work. Once Selected, Then Auto. Primary Contact Name & Email displayed.

Tasks:-
1. HTML Prototype of Dashboard & RFI.
2. Review by Team.
3. Integration with Codebase.

**Comment by vikasagarwal** [comment]
@amit-k86 In this story, are we talking about the search a user performs for finding a business? Or, are we allowing search only to find a specific contact? Or, are we allowing search of PO's/SO's/RFI's etc. that the searching user's company has ever transacted using?

Please supply details.

**Comment by amit-k86** [comment]
@vikasagarwal Currently just taking about Search in Dashboard. Search in Dashboard to be made for business. updated above to the story.

**Comment by priyankshukla** [comment]
Agreed Amit, search in Dashboard to be made for business but I think, we should include the nature of business (category) in this too.

**Comment by amit-k86** [comment]
@priyankshukla - Agreed.!! In Search - Business & nature of biz search to be there.

Another 2 points (To Be Discussed):-
1. Biz to be allowed to be Private so that not shown in search.
2. Biz can upload there catalogue on Black Label. Once searched. person can see catalogue of services and products offered.

**Comment by vikasagarwal** [comment]
@amit-k86, These two are two new features. Let's discuss these in our next
meeting and we can created stories for them then.

On Sun, Mar 22, 2015 at 3:01 PM, amit-k86 <notifications@github.com> wrote:

> @priyankshukla <https://github.com/priyankshukla> - Agreed.!! In Search -
> Business & nature of biz search to be there.
>
> Another 2 points (To Be Discussed):-
> 1. Biz to be allowed to be Private so that not shown in search.
> 2. Biz can upload there catalogue on Black Label. Once searched. person
> can see catalogue of services and products offered.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OrderMail/blacklabel/issues/22#issuecomment-84575171>.
>



-- 
Best Cricket T20 Coverage on:
http://cricket-t20.com
